src/iterated_integrals.jl

```diff
@@ -186,10 +186,11 @@ function get_iterated_I!(dt, dW, Wik::WikJGeneral_oop, C=1)
     sum_dW² = dot(dW,dW)
 
     WikJ = dW*dW'
-    Gp₁ = randn(M)
-    α = sqrt(1 + sum_dW²/dt)
-    Gp₂ = Gp₁/(sqrt(2)*(1+α)*dt)
-
+    eltWikJ = eltype(WikJ)
+    WikJ2 =zeros(eltWikJ, size(WikJ))
+    Gp₁ = randn(eltype(WikJ),M)
+    α = eltype(WikJ)(sqrt(1 + sum_dW²/dt))
```

Comment:
use convert instead of implicit number constructor

Reply:
ok